modules/swagger-parser/src/main/java/io/swagger/parser/SwaggerResolver.java

```diff
@@ -49,6 +50,16 @@ public SwaggerResolver(Swagger swagger, List<AuthorizationValue> auths, String p
         parametersProcessor = new ParameterProcessor(cache, swagger);
     }
 
+    public SwaggerResolver(Swagger swagger, List<AuthorizationValue> auths, String parentFileLocation, Settings settings, ParseOptions parseOptions) {
```

Comment:
instead of duplicating logic in constructor I would suggest to call in constructor above this(swagger, auths, parentFileLocation, settings, new ParseOptions())